app/controllers/application_controller.rb

```diff
@@ -7,6 +7,9 @@ class ApplicationController < ActionController::API
   class Unauthorized < RuntimeError
   end
 
+  class NotImplemented < Exceptions::NotImplemented
+  end
+
```

Comment:
I don't think we need this shortcut name.  Just refer to it as `Exceptions::NotImplemented`